app/assets/javascripts/components/meetings_list.es6.jsx

```diff
@@ -0,0 +1,68 @@
+class MeetingsList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.perPage = 12;
```

Comment:
Well... I'm not sure about including this on the component's state but I think it's cleaner to include it as a prop. What do you think?